### DISCUSSION TITLE
SHORTCUTS

### DISCUSSION THREAD

**Discussion by Lunaire333** [AmyE123/TactiKit#10] [Map Editor Ideas]
I don't know if it will be in the toolkit, but having shortcuts for each tool present in the software would be greatly appreciated and would make the user spend less time clicking this or that when designing/editing their map.

![image](https://github.com/AmyE123/TactiKit/assets/158181584/043ae01b-5aad-4f76-b3af-b130b25a5af3)

**Comment by AmyE123** [comment]
Nice idea. I like this!
I'll have to look up what other software uses for the shortcut keys, any suggestions?

**Reply by Lunaire333** [reply]
maybe having shortcuts of most of the buttons/functions present in the Toolkit would be great.

**Comment by AmyE123** [comment]
Relation to: https://github.com/AmyE123/TactiKit/discussions/16 as this discusses shortcuts!
